docs/components/text.md

```diff
@@ -166,6 +166,29 @@ the SDF font, DejaVu:
 [fontexampleimage]: http://i.imgur.com/iWtXHm5.png
 ![Arial Black vs. Deja Vu][fontexampleimage]
 
+### Non-ASCII Characters
+
+To use non-ascii characters, you need to create your own custom font. The easiest way is to use [a web-based MSDF tool][msdf-bmfont-web]. Select your required character set and generate your own custom msdf font zip file.
+
+Once you download your custom msdf font zip file, extract it and rename `*.png` to `*-msdf.png`, then put both png and json files to your A-Frame directory.
```

Comment:
Hm do we know why the PNG needs to be renamed? The filename is listed inside the JSON file; I'm surprised by this. 🤔

Reply:
That is because text component loads an image named fontName-msdf.png by default. It does not use image filename in json.

See https://github.com/aframevr/aframe/blob/dcdc1235600202c8c0b1f7ac0df21a8cc4018ef6/src/components/text.js#L253

Reply:
Should we fix text component?